### PR TITLE
Address security alert

### PR DESCRIPTION
**Pull request by FokkeZB**
See https://github.com/zapier/zapier-platform-schema/network/alerts

As part of my T3 I thought I'd address some of these in my mailbox.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "jsonschema": "1.1.1",
-    "lodash": "4.17.10"
+    "lodash": "^4.17.11"
```
Comment (xavdid) [inline package.json]: I'd remove the `^` as it's important that we know exactly what version of lodash goes out. not that I expect lodash to break anything or do anything weird, but it ensures we're in control of what goes out.

**Review by xavdid — APPROVED** [review]
approving as not to block you, but I strongly suggest requiring an exact version before merging.

File: package.json (hunk 1 above)
Comment (FokkeZB) [reply package.json]: IMHO that's what you have a lock-file for, but I'll drop it :)

**Comment by xavdid** [comment]
The lockfile ensures developers all work with the same version, but the lockfile isn't published to the end-user. Info here: https://docs.npmjs.com/files/package-lock.json

**Comment by FokkeZB** [comment]
O wow, I always thought it guaranteed dependencies for end-user installs as well! I see that's what shrinkwrap does.

I see the same is true for `yarn.lock` and they actually explain why very well at:
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

> First, it’s not going to be every user that is affected by this. It’s well agreed upon that applications should be using lockfiles, and if they are then they won’t be affected by sudden breaking changes.

TOL!

**Pull request merged**